docs/internals/variable_cleanup.rst

```diff
@@ -1,31 +1,36 @@
 .. index: variable cleanup
 
-*********************
-Cleaning Up Variables
-*********************
+*************************
+Cleaning of Padding Bits
```

Comment:
In my view, "padding" is only something that is used in byte arrays, i.e. data structures of dynamic length.